torch/export/_draft_export.py

```diff
@@ -252,10 +253,11 @@ def draft_export(
                 log_contents["stack"] = filter_stack(
                     log_contents["stack"], str_to_filename
                 )
-                if hash_stack(log_contents["stack"]) in data_dependent_logs:
+                data_dependent_logs[hash_stack(log_contents["stack"])] += 1
```

Comment:
I'm not sure if this is an accurate way to count right now 😅 but also maybe I should fix how I dedup things. 
The hash is based on just the stacktrace, so if theres multiple data dependent errors that occur on that one line, I only log the first one... we can discuss better ways to dedup.